Orange/data/sql/table.py

```diff
@@ -654,3 +654,19 @@ def _execute_sql_query(self, query, param=None):
 
     def checksum(self, include_metas=True):
         return np.nan
+
+    def __get_nan_frequency(self, columns):
+        nans = 0
+        for column in columns:
+            col_field = column.to_sql()
+            query = self._sql_query(["COUNT(*)"],
+                                    filters=[f"{col_field} IS NULL OR {col_field} = 'NaN'"])
```

Comment:
Checking NULL is sufficient. Checking `"nan"` even fail on MSSQL when column is float.
```suggestion
        query = self._sql_query(["COUNT(*)"], filters=[f"{col_field} IS NULL"])
```

Reply:
Anyway, can we make it in one query to speed the process up? For example with this: https://stackoverflow.com/a/16528738/3551700